pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[tool.black]
```

Comment:
Should this file be checked in? Looks like an editor file.

Reply:
This is going to be used for code formatting checks using the [Python Black ](https://github.com/psf/black) formatter. It is added in requirement/dev.txt.
I didn't run it yet because it will cause a bunch of small changes to fix code style that I didn't want part of this PR.

But yes, it should be checked in.